vnext/ReactUWP/Views/TextInputViewManager.cpp

```diff
@@ -280,6 +281,28 @@ void TextInputShadowNode::registerEvents() {
         }
       });
 
+  m_controlKeyDownRevoker = control.KeyDown(
+      winrt::auto_revoke, [=](auto &&, winrt::KeyRoutedEventArgs const &args) {
+        if (args.Key() == winrt::VirtualKey::Enter && !args.Handled()) {
+          auto instance = wkinstance.lock();
+          folly::dynamic eventDataSubmitEditing = {};
```

Comment:
>eventDataSubmitEditing [](start = 25, length = 22)

The event has three parameters: {nativeEvent: {text, eventCount, target}}. I don't know if android/iOS provides eventCount or not.

Reply:
We also don't send eventCount for onEndEidting event, I want to keep the behavior same for onSubmitEditing for now.

---
In reply to: [335124057](https://github.com/microsoft/react-native-windows/pull/3423#discussion_r335124057) [](ancestors = 335124057)